src/controllers/legacycontrollermapping.h

```diff
@@ -38,10 +38,10 @@ class LegacyControllerMapping {
               m_scripts(other.m_scripts),
               m_deviceDirection(other.m_deviceDirection) {
     }
+    LegacyControllerMapping& operator=(const LegacyControllerMapping&) = delete;
```

Comment:
rule of 5. lets also define the move operations.

Reply:
Right, but the rule of five primarily says that you should explicit about it instead of rely on the complicated rules on whether they're autogenerated or not. 
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-copy-move-or-destructor-function-define-or-delete-them-all